src/redux/books/books.js

```diff
@@ -0,0 +1,32 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD_BOOK = 'bookstore-react/book/ADD_BOOK';
+const DELETE_BOOK = 'bookstore-react/book/DELETE_BOOK';
```

Comment:
- [ ] I think 🤔  from this step, you should now configure the Redux Store and write actions and reducers for adding and removing books, but I have yet to add or remove any books from the list. Please do this.😅 👍🏻 